database/migrations/2021_11_02_230827_create_transactions_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTransactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('transactions', function (Blueprint $table) {
+            $table->id('transaction_id');
+            $table->enum('type', ['income', 'outcome']);
```

Comment:
If its enum, i think you should be make this uppercase

Reply:
OK